Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CItemAchievementRewardReceiveNtc.cs

```diff
@@ -0,0 +1,42 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Model;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure;
+
+/// <summary>
+///     This NTC has to be sent after an achievement's receivable reward has been clicked on in the achievement UI and the
+///     player should unlock a new craft recipe.
+/// </summary>
+public class S2CItemAchievementRewardReceiveNtc : IPacketStructure
+{
+    public PacketId Id => PacketId.S2C_ITEM_ACHIEVEMENT_REWARD_RECEIVE_NTC;
+    /// This could also be reward type..
+    public ItemNoticeType UpdateType;
```

Comment:
Are you sure it shares its values with the item notice?

Reply:
You are right, it conveniently fit the sizes, but after playing around none of these values seem to have any effect on the UI, except for ItemId. Renamed all to Unk* for now.